### ISSUE TITLE
Default Autoupdate setFeedURL will not work with electron-release-server for Windows

### ISSUE THREAD

**Issue by ethanwillis** [wojtkowiak/meteor-desktop#69]
https://github.com/wojtkowiak/meteor-desktop/blob/07d7a005e5a6b47c14addca4cbe3d55723cb7ff9/skeleton/squirrel.js#L124

Starting at this line we need to modify the block to generate URLs that are valid for both the Windows and OSX update URL formats for electron release server.

For example. In the current meteor-desktop default settings we pass an autoUpdateFeedUrl of "127.0.0.1:3000/update/:platform/:version" This will not work with Windows Update URL in electron-release-server [1]. It is looking for a feed URL of the form 127.0.0.1:3000/update/:platform/:version/RELEASES.

However the default settings for our autoupdate feed url do work for electron-release-server's OS X format of 127.0.0.1:3000/update/:platform/:version[/:channel]. See [2]

[1] https://github.com/ArekSredzki/electron-release-server/blob/master/docs/update-windows.md
[2] https://github.com/ArekSredzki/electron-release-server/blob/master/docs/update-osx.md

**Comment by ethanwillis** [comment]
Upon further investigation electron-release-server is the issue. It is not fully compatible with squirrel.windows. This is because when squirrel.windows tries to download a release file electron-release-server gives it a URL with query parameters and squirrel.windows is expecting either a static HTTP[S] URL with no query parameters. Or it is expecting an absolute file name.


I am going to try this: https://github.com/Aluxian/squirrel-updates-server and then I'll report back. If it works I'll open a PR on the readme for this project explaining that electron-release-server doesn't currently work and I'll suggest squirrel-updates-server

**Comment by wojtkowiak** [comment]
Hey,
got only a moment to respond. I have only used https://github.com/ArekSredzki/electron-release-server and tested it on both Windows and Mac so it somehow worked for me with this `http://127.0.0.1/update/:platform/:version`. It has been a while so I might be missing something now. 
The pitty is that I did not have enough time to make integration tests for that as it would be very complicated to setup a server and test it all on a public CIs (they are very resource limited)

I currently have a plan to investigate this and maybe only support `electron-updater`. `electron-builder` now recommends packaging to `nsis` instead of `squirrel` and nsis has its own auto update mechanism.

Anyway, I want to move the auto update support out of the core and only make plugins to support each mechanisms individually.
